.travis.yml

```diff
@@ -11,11 +11,14 @@ jobs:
     # Basic Checks
     - stage: primary
       env: TOXENV=docs
+      python: 3.6
     - env: TOXENV=lint-py2
     - env: TOXENV=lint-py3
       python: 3.6
     - env: TOXENV=mypy
+      python: 3.6
     - env: TOXENV=packaging
+      python: 3.6
```

Comment:
I ran into an issue with the mypy job after upgrading tox / the testsuite requirements, upgrading the Python version fixed it. Since there's no reason to stay with Python 2.7 for the docs and packaging jobs, I upgraded those too.

Reply:
OK, while we're on the subject of the linter, do we really need the 2 jobs? Is there any point in running `lint-py2`?

Reply:
IIRC, it's because flake8 gives different results when run on Py2 vs Py3 and we've had situations where flake8 failed for Py3 but not Py2 on Travis.